scripts/prepack.ts

```diff
@@ -69,28 +99,54 @@ if (tmpCopyBundles) {
 }
 // end remove
 
-// package.json modifications
-const packageJsonPath = path.resolve(buildDir, 'package.json');
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const pkgJson: { [key: string]: unknown } = require(packageJsonPath);
+const packageJsonPath = path.resolve('package.json');
+const pkgJson: { [key: string]: unknown } = JSON.parse(fs.readFileSync(packageJsonPath, { encoding: 'utf8' }));
+
+function getModifiedPkgJson(newName: string, removeDependencies?: boolean): Record<string, unknown> {
+  const newPkgJson = { ...pkgJson };
+  newPkgJson.name = newName;
 
-// modify entry points to point to correct paths (i.e. strip out the build directory)
-ENTRY_POINTS.filter(entryPoint => pkgJson[entryPoint]).forEach(entryPoint => {
-  pkgJson[entryPoint] = (pkgJson[entryPoint] as string).replace(`${buildDir}/`, '');
-});
+  delete newPkgJson.scripts;
+  delete newPkgJson.volta;
+  delete newPkgJson.jest;
 
-delete pkgJson.scripts;
-delete pkgJson.volta;
-delete pkgJson.jest;
+  if (removeDependencies) {
+    newPkgJson.dependencies = {};
```

Comment:
Just a question: Can we do this? I mean for the legacy build we can remove the `@sentry` dependencies but what about `tslib` and `localforage`? Probably possible but I wanted to double-check.

Reply:
This is valid. Updated here: 443f7460c3ae12c5a06f2d9cd6017dfbdd5e7ee9

Re-requesting review